modoboa/core/extensions.py

```diff
@@ -118,13 +118,14 @@ def load_all(self):
     def list_all(self, check_new_versions=False):
         """List all defined extensions."""
         result = []
+        new_extensions = {}
         if check_new_versions:
-            new_extensions = ModoAPIClient().list_extensions()
-            new_extensions = dict(
-                (ext["name"], ext["version"]) for ext in new_extensions
-            )
-        else:
-            new_extensions = {}
+            tmp = ModoAPIClient().list_extensions()
```

Comment:
Could you rename tmp to something more verbose (like new_extensions as before) ?